packages/application-shell/src/index.js

```diff
@@ -27,3 +27,14 @@ export {
 } from './utils/setup-global-error-listener';
 export { selectUserId, selectProjectKeyFromUrl } from './utils';
 export { default as AsyncChunkLoader } from './components/async-chunk-loader';
+
+/**
+ * NOTE:
+ *    These are library re-exports which are shared between app-kit and
+ *    MC apps. Not having them re-exported here often leads to bundling
+ *    them twice given app-kit and any MC app require different versions.
+ *    Something they never should anyways and causes breakage.
+ *
+ *    More information can be found here: https://github.com/commercetools/fe-chapter-notes/issues/61
```

Comment:
so we link to our FE-chapter on OSS? 🤔

Reply:
Why not. It will show a 404 anyway. It doesn't contain a ticket number of anything. I wouldn't worry too much about it.